columnflow/tasks/ml.py

```diff
@@ -96,10 +125,14 @@ def requires(self):
     @MergeReducedEventsUser.maybe_dummy
     def output(self):
         k = self.ml_model_inst.folds
-        return {"mlevents": law.SiblingFileCollection([
-            self.target(f"mlevents_fold{f}of{k}_{self.branch}.parquet")
-            for f in range(k)
-        ])}
+        outputs = {
+            "mlevents": law.SiblingFileCollection([
+                self.target(f"mlevents_fold{f}of{k}_{self.branch}.parquet")
+                for f in range(k)
+            ]),
+            "stats": self.target(f"stats_{self.branch}.parquet"),
```

Comment:
Can there be a "stats" output if there is no preparation producer? If not, there should be no "stats" entry at all I assume.

Reply:
Currently, there is one default field that is produced independently of the preparation_producer: https://github.com/columnflow/columnflow/blob/5c5dbf2edf76787968384a0fd65559d448a0103f/columnflow/tasks/ml.py#L251-L253
In that way, we always have some stats file that can be merged. If we change that, we also have to make sure that MergeMLStats will not be requested by MLTraining

Reply:
I think this is actually a field one always wants / needs to have, so I'm good with that :)